### ISSUE TITLE
$POE_ROOT/$POE_PWD unavailable in tool.poe.env section

### ISSUE THREAD

**Issue by tumidi** [nat-n/poethepoet#198]
I'd expect `$POE_ROOT`/`$POE_PWD` variables to get expanded in `[tool.poe.env]`.

```toml
[tool.poe.env]
GLOBAL_POE_ROOT = "${POE_ROOT}"  # doesn't work

[tool.poe.tasks.my-task]
default_item_type = "cmd"
sequence = [
  "echo GLOBAL_POE_ROOT=${GLOBAL_POE_ROOT}",
  "echo TASK_POE_ROOT=${TASK_POE_ROOT}",
]

[tool.poe.tasks.my-task.env]
TASK_POE_ROOT = "${POE_ROOT}"  # works
```

I created a minimal reproduction: https://github.com/tumidi/poethepoet-env-test

Output is:

```sh
$ poe my-task
GLOBAL_POE_ROOT=
TASK_POE_ROOT=/tmp/poethepoet-env-test
```

**Comment by nat-n** [comment]
Hi @tumidi, yes I would expect that too. Thanks for reporting this.

**Comment by nat-n** [comment]
This was incidentally fixed by the recent  refactor of config management that is now on the development branch.

**Comment by nat-n** [comment]
This issue is now fixed with [0.26.0](https://pypi.org/project/poethepoet/0.26.0/) 🚀
